test/e2e/utils.ts

```diff
@@ -26,6 +26,22 @@ export const createLocalAthensDB = async (page:Page, dbName:string) => {
     await page.pause()
 };
 
+export const getTitle = async (page:Page) => {
+    const titleLocator = await page.locator("h1 >> nth=0");
+    const titleText = await titleLocator.innerText();
+    return titleText;
+};
+
+export const todaysDate = async (page:Page) => {
+    const todaysDate = await page.evaluate(`athens.dates.get_day()`);
```

Comment:
Wasn't sure how to utilize ClojureScript lib from node.js (maybe some shadow stuff). This seemed easier.

Reply:
Seems like calling code from the console doesn't work in CI anyways. Is it bc it's headless? https://github.com/athensresearch/athens/pull/2026/checks